Tests/CodeGenerateTests/CodeGenerateTests.swift

```diff
@@ -1,17 +1,97 @@
 import XCTest
-@testable import SmokeFrameworkApplicationGenerate
-
+@testable import SmokeFrameworkApplicationGenerateCommand
 class CodeGenerateTests: XCTestCase {
-    func testExample() {
-        print("This is an example test")
+
+    func testExecutionPlanServiceGenerate() {
+        let swaggerFilePath = "/Users/mndzup/Desktop/Service_Swagger/ExecutionPlanService"
```

Comment:
Is this a local file path?  If so can we instead place these files in the repo itself?